README.md

```diff
@@ -1,111 +1,175 @@
 # ⑂ forker
 
+[![Version](https://img.shields.io/badge/Version-0.0.1-7F187F.svg)](https://github.com/lelia/forker/releases)
+[![License: MIT](https://img.shields.io/badge/License-MIT-7462E0.svg)](LICENSE)
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.0-24B8EE.svg)](CODE_OF_CONDUCT.md)
 [![Tests](https://github.com/lelia/forker/actions/workflows/test.yml/badge.svg)](https://github.com/lelia/forker/actions/workflows/test.yml)
 
-Github action to automate fork creation. This action uses [octokit.js](https://github.com/octokit/octokit.js) and the [GitHub API](https://docs.github.com/en/rest) to automatically create a repository fork, either in your personal namespace or an organization you administer. 
+Github action to automate fork creation. This action uses [octokit.js](https://github.com/octokit/octokit.js) and the [GitHub API](https://docs.github.com/en/rest) to automatically create a repository fork, either in your personal namespace or an organization you administer.
```

Comment:
Super minor nitpick here, but should the casing of GitHub be standardized throughout the documentation?

Reply:
Yep, definitely should be standardized (it's a pet peeve of mine, as well, but I didn't think to check the portions of the README that were auto-generated from the Actions template 😛)

Reply:
Updated all `Github` refs to `GitHub` that I could find. LMK if I missed any.